client/components/clipboard-button-input/README.md

```diff
@@ -22,22 +22,7 @@ export default function MyComponent() {
 
 The following props can be passed to the ClipboardButtonInput component. With the exception of `className`, all props, including those not listed below, will be passed to the child `<input />` element.
 
-### `value`
-
-<table>
-	<tr><td>Type</td><td>String</td></tr>
-	<tr><td>Required</td><td>No</td></tr>
-	<tr><td>Default</td><td><code>""</code></td></tr>
-</table>
-
-The value of the `<input />` element, and the text to be copied when clicking the copy button.
-
-### `disabled`
-
-<table>
-	<tr><td>Type</td><td>Boolean</td></tr>
-	<tr><td>Required</td><td>No</td></tr>
-	<tr><td>Default</td><td><code>false</code></td></tr>
-</table>
-
-Whether the children `<input />` and `<button />` should be rendered as `disabled`.
+| prop     | type    | required | default | comment |
```

Comment:
The nice thing about introducing the markdown parser, is that we don't have to change any existing readmes (or at least a few of them). I mostly did this to get some good table styles, I'll likely remove these commits.

I have the parser sanitizing html instead of rendering it to avoid the possibility of someone injecting script tags via a readme.

I'd like to open a different PR to convert all the readme `<table />` tags to markdown tables so that they can be easily read without a browser.